inscriptions/ISic001382.xml

```diff
@@ -193,36 +198,24 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI140888">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#TNS">
                 <ab>
-                    <lb n="1"/>
-                    <supplied reason="lost">— — ὁ δεῖνα — —</supplied>
-                    <lb n="2"/>
-                    <supplied reason="lost">ζή</supplied>σας ἔ<supplied reason="lost">τη —ʹ· ἐνθά</supplied>-
-                    <lb n="3"/>
-                    <supplied reason="lost">δε</supplied>
-                    κῖτε ἐν <supplied reason="lost">εἰρή</supplied>-
-                    <lb n="4"/>
-                    <supplied reason="lost">νῃ</supplied>, ἀγορά<supplied reason="lost">σας</supplied>
-                    <surplus>²⁷ἀγορα[σάμενος]</surplus>²⁷
-                    <lb n="5"/>
-                    <supplied reason="lost">τὸν</supplied>
-                    τόπον
-                    <supplied reason="lost">ἑαυτῷ</supplied>
-                    <lb n="6"/>
-                    <supplied reason="lost">καὶ τοῖς ἰδίοις</supplied>.
-                    <lb n="7"/>
+                    <lb n="1"/><supplied reason="lost">ὁ δεῖνα</supplied><gap reason="lost" quantity="1" unit="line"/>
```

Comment:
instead of ὁ δεῖνα or equivalent, code a `<gap>` and inside the gap put a `<desc>personal name</desc>`